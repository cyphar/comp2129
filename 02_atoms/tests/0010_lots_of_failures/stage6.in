SAVE hello.as
SAVE whatisgoingon
SAVE
SAVE A A A
SAVE   a a  aa aa
START 2 3 1
SAVE a aaa aaa a
SAVE
SAVE a.a
SAVE lol.a a
SAVE a.a
SAVE a.a
SAVE a.a
