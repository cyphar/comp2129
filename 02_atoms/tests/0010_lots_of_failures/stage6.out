Invalid Command

Invalid Command

Invalid Command

Invalid Command

Invalid Command

Game Ready
Red's Turn

Too Many Arguments

Missing Argument

Game Saved

Too Many Arguments

File Already Exists

File Already Exists

File Already Exists

